### DISCUSSION TITLE
How to mesh with element set as groups

### DISCUSSION THREAD

**Discussion by Kullapong** [neperfepx/neper#518] [Q&A] [answered]
I'm trying to generate microstructure of Dual Phase steel with Neper, I had success with creating .tess by use -group function to group it with id and visualize it with -V module but I can't meshing it with a specific element set as set 1 for group 1 and set 2 for group 2 in order to use as input file to perform RVE simulation on Abaqus . 
Is there any method that I can mesh it like that on Neper?.

The code that I used are below:
```
neper -T -n 1600 -domain "square(40,40)" -id 1 -dim 2 -morpho\
 "diameq:lognormal(2,0.35),1-circularity:lognormal(0.02,0.1)"\
 -morphooptistop "time=3" -group "id<160?1:2"\
 -reg 1  -o DP_test
 ```

![DP_test2](https://user-images.githubusercontent.com/109538813/179552694-cc44b2fe-6032-4968-a4a3-85aa2b501fec.png)

**Comment by rquey** [comment, marked as answer]
Writing elsets for phases is sort of gray area for Neper, as it widely assumes that elsets represent individual grains. I understand that, in your simulation, all grains of the same group/phase will be assigned the same behavior.  Couldn't you just keep the elsets as they are and assign the same material behavior to all elsets/grains of the same group/phase?

**Reply by ashokambig** [reply]
Dear Quey,

I am also struck here. 
for your question : **"Couldn't you just keep the elsets as they are and assign the same material behavior to all elsets/grains of the same group/phase?"-** for me this will work.. If I know the elsets of my group (i.e phase) I can easily assign the needed behaviour to these group.
But how to create an abaqus input (=inp file) file that has the groups created from the needed elsets?. Below is the image from the abaqus. I thought you will show me an option to create some items under the disply group shown in the image here.

Thanks for your help

Regards
Ashok

![group](https://user-images.githubusercontent.com/91887761/205653208-262cf6da-289d-49fc-833d-211c48f6807f.PNG)
